exercises/e001/results/mikjarz/.gitignore

```diff
@@ -0,0 +1,2 @@
+    .DS_Store
+    .idea/
```

Comment:
I think this file should be added / modified at the top of the repo. Your `.idea` files were added at the top of the repo.

Reply:
Could you remove those files and create `gitignore(5)` to exclude them in general?